07-Template-Distribution-Protocol.md

```diff
@@ -24,7 +34,17 @@ Further, the Template Provider MUST consider the maximum additional bytes requir
 | ----------------------------------- | --------- | ----------------------------------------------------------------------------------------------- |
 | coinbase_output_max_additional_size | U32       | The maximum additional serialized bytes which the pool will add in coinbase transaction outputs |
 
-## 7.2 `NewTemplate` (Server -> Client)
+## 7.3 `SetNonce` (Client -> Server)
+
+If client is a `JobDeclarator`, it need to receive a tx short hash list of the transactions that
+are in the block candidate for each `NewTemplate`. In order to calculate the list the server and the
+client needs negotiate a nonce.
```

Comment:
is the "nonce" used to hash for the short hash list ? Is it used for every new template ? Because in that case the name "nonce" may be a little bit misleading IMO

Reply:
right, we could call it `tx_short_hash_list_nonce` what do you think?